holidayTable.ts

```diff
@@ -0,0 +1,2922 @@
+interface HolidaysDays {
+    swi_Data: string;
```

Comment:
w sumie jak resztę kodu masz po angielsku, to tutaj też właściwie mógłbyś zmienić

Reply:
poprawiłem